Cookbook/Proposals/MocksAndStubsNextToImplementation.md

```diff
@@ -0,0 +1,53 @@
+# Put Mocks, Stubs Next to Implementation
+
+* Author(s): Sergey Shulga
+* Review Manager: TBD
+
+## Introduction
+
+Put implementation of mocks next to the implementation of the protocol
+
+## Motivation
+
+
+- Improves Discoverability of mocks
```

Comment:
Can we not follow the pattern "{ModelName}+Fixture.swift" and put it in Test utilities.

Reply:
This way we can easily find it by filtering out this name in project navigator

Reply:
I read some comments on conversation tab, and I see why we don't want to put all of them in test utilities framework